compiler/compile/ResolvedMethod.cpp

```diff
@@ -40,258 +40,13 @@ class TR_PrexArgInfo;
-
-       // DAA Packed Decimal shift methods
-       getRecognizedMethod() == TR::com_ibm_dataaccess_PackedDecimal_shiftLeftPackedDecimal_  ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_PackedDecimal_shiftRightPackedDecimal_ ||
-
-       // DAA Packed Decimal check method
-       getRecognizedMethod() == TR::com_ibm_dataaccess_PackedDecimal_checkPackedDecimal_ ||
-
-       // DAA Packed Decimal <-> Integer
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertPackedDecimalToInteger_   ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertPackedDecimalToInteger_ByteBuffer_ ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertIntegerToPackedDecimal_   ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertIntegerToPackedDecimal_ByteBuffer_ ||
-
-       // DAA Packed Decimal <-> Long
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertPackedDecimalToLong_      ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertPackedDecimalToLong_ByteBuffer_      ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertLongToPackedDecimal_      ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertLongToPackedDecimal_ByteBuffer_      ||
-
-       // DAA Packed Decimal <-> External Decimal
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertExternalDecimalToPackedDecimal_ ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertPackedDecimalToExternalDecimal_ ||
-
-       // DAA Packed Decimal <-> Unicode Decimal
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertPackedDecimalToUnicodeDecimal_  ||
-       getRecognizedMethod() == TR::com_ibm_dataaccess_DecimalData_convertUnicodeDecimalToPackedDecimal_)
-      {
-      return true;
-      }
-   return false;
-#else
-   return false;
-#endif
-   }
-
-bool TR_ResolvedMethod::isDAAIntrinsicMethod()
-   {
-#ifdef J9_PROJECT_SPECIFIC
-   return isDAAMarshallingIntrinsicMethod() || isDAAPackedDecimalIntrinsicMethod();
-#else
-   return false;
-#endif
-   }
-
-TR::Method *TR_ResolvedMethod::convertToMethod()                          { TR_UNIMPLEMENTED(); return 0; }
+bool         TR_ResolvedMethod::isDAAMarshallingWrapperMethod()            { TR_UNIMPLEMENTED(); return false; }
+bool         TR_ResolvedMethod::isDAAPackedDecimalWrapperMethod()          { TR_UNIMPLEMENTED(); return false; }
+bool         TR_ResolvedMethod::isDAAWrapperMethod()                       { TR_UNIMPLEMENTED(); return false; }
+bool         TR_ResolvedMethod::isDAAMarshallingIntrinsicMethod()          { TR_UNIMPLEMENTED(); return false; }
+bool         TR_ResolvedMethod::isDAAPackedDecimalIntrinsicMethod()        { TR_UNIMPLEMENTED(); return false; }
+bool         TR_ResolvedMethod::isDAAIntrinsicMethod()                     { TR_UNIMPLEMENTED(); return false; }
```

Comment:
Unless I'm misreading the code, I don't think it is necessary to keep these 6 DAA functions in OMR.  They seem to be either only called from OpenJ9 or called amongst themselves.

Reply:
Ugh

Reply:
All six of those functions are only used in a very limited number of places (8, I believe) in either EstimateCodeSize.cpp or InlinerTempForJ9.cpp, so if you have to introduce casts I think the effect will be minimal.

Reply:
Addressed in [f533c19](https://github.com/eclipse-omr/omr/commit/f533c190f38f3193aafbf5c5a1600f0ad6354dfa). Ready for another review. Thank you!